core/blockchain_test.go

```diff
@@ -163,14 +163,14 @@ func testBlockChainImport(chain types.Blocks, blockchain *BlockChain) error {
 		if err != nil {
 			return err
 		}
-		receipts, _, usedGas, err := blockchain.processor.Process(block, statedb, vm.Config{})
+		res, err := blockchain.processor.Process(block, statedb, vm.Config{})
 		if err != nil {
-			blockchain.reportBlock(block, receipts, err)
+			blockchain.reportBlock(block, res.Receipts, err)
```

Comment:
If an error occurs in process, this would never run as the function would return early. It seems reasonable to check if `res` is `nil` inside `reportBlock` though, so I will update it.